src/components/Thread.vue

```diff
@@ -247,8 +248,8 @@ export default {
 				this.loading = false
 			} catch (error) {
 				logger.error('could not load envelope thread', { threadId, error })
-				if (error.isError) {
-					this.errorMessage = t('mail', 'Could not load your message thread')
+				if (error?.response?.status === 403) {
+					showError(t('mail', 'Could not load your message thread, it doesnt exist or is deleted'))
 					this.error = error
 					this.loading = false
 				}
```

Comment:
add an `else` for any other error and show *Could not load your message thread*

that could happen if the network is down, etc.